server/server.js

```diff
@@ -257,22 +253,8 @@ server.set('port', port);
 
 server.disable('x-powered-by');
 server.use(helmet());
-server.set('views', `${__dirname}/views`);
-server.set('view engine', 'mustache');
-server.engine('html', mustacheExpress());
-
-const renderSøk = () =>
-    new Promise((resolve, reject) => {
-        server.render('index.html', miljøvariablerTilFrontend, (err, html) => {
-            if (err) {
-                reject(err);
-            } else {
-                resolve(html);
-            }
-        });
-    });
```

Comment:
Nå bygger vi index-fila med webpack og HtmlWebpackPlugin, så trenger ikke mustache på serveren. For å være helt ærlig, skjønner jeg ikke hvorfor vi sendte inn `miljøvariablerTilFrontend` her, men skal fungere slik vi gjør det nå:

1) Skriver miljøvariabler til fil (`writeEnvironmentVariablesToFile()`) når vi kjører opp serveren
2) Importerer denne fila fra index.html (import legges inn fra webpack)